readme.adoc

```diff
@@ -251,6 +251,22 @@ terraform output config-map-aws-auth > aws-auth.yaml
 With this file and the `kubeconfig` file out you can then configure `kubectl` to
 use the `kubeconfig` file and apply the `aws-auth` configmap.
 
+If you plan to use link:https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler[`cluster-autoscaler`], a `values.yaml`-file can be created as well 
+by using the `output` subcommand. 
+
+[source,shell]
+----
+terraform output config-map-aws-auth > aws-auth.yaml
```

Comment:
I think this should be `terraform output cluster-autoscaler > values.yaml` correct?